src/cache/s3.rs

```diff
@@ -29,9 +29,7 @@ impl S3Cache {
     ) -> Result<Operator> {
         let mut builder = S3::default();
         builder.bucket(bucket);
-        if let Some(region) = region {
-            builder.region(region);
-        }
+        builder.region(region.unwrap_or("us-east-1"));
```

Comment:
Until now, OpenDAL would take `us-east-1` as default in case of a missing region, but this is now an error.
This change makes it consistent with the current behavior, as I suspect a lot of users rely on the current
way of doing things.
```suggestion
        builder.region(region.unwrap_or("us-east-1"));
```

Reply:
> Where should I log this breaking change?

I don't think it's a breaking change, it's a bug fix.